cf-custom-resources/lib/trigger-state-machine.js

```diff
@@ -0,0 +1,135 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+"use strict";
+
+const aws = require("aws-sdk");
+
+/**
+ * Main handler, invoked by Lambda
+ */
+exports.handler = async function (event, context) {
+  const responseData = {};
+  const physicalResourceId = event.PhysicalResourceId || event.LogicalResourceId;
```

Comment:
I see - we don't ever ever want to change `physicalResourceId` right? In this case, what about we just do `const physicalResourceId = "trigger-state-machine"`

Reply:
my thought here was - if we ever reuse this custom resource in the same lambda, we'd have two different logical id's with the same physical id, whereas this gives us that flexibility+easy backwards compatibility (even if we change the logical id, we wouldn't accidentally break existing services)...not sure if this is something to actually be concerned about though?